lib/conjur/cli.rb

```diff
@@ -91,6 +93,27 @@ def init!
         load_plugins
         commands_from 'conjur/command'
       end
+
+      def command_version_compatible? command
```

Comment:
Harder to read, and also wrong (the version test was backwards). :)

How about this:

```
  def command_version_compatible? command
    !command.instance_variable_defined?(:@conjur_min_version) ||
      (appliance_version &&
        command.instance_variable_get(:@conjur_min_version) <= appliance_version
      )
  end
```